runtime/vm/visible.c

```diff
@@ -29,6 +29,70 @@
 #include "util_api.h"
 #include "vm_internal.h"
 
+
+
+/**
```

Comment:
Thanks for expanding the comment.  You've got all the necessary info here.  I think it would be easier to read if formatted slightly differently:
```
* Check module access from srcModule to destModule. 
*
* The algorithm for validating module access is:
* 1) check to see if the source module `reads` the dest module. 
* 2) check to see if the dest module exports the package (that 
* dest class belongs to) to the source module. 
*
* For reflective calls, the rules are slightly different as all reflect
* reflect accesses implicitly have read access.  Set the 
*  J9_LOOK_REFLECT_CALL flag in the lookup options for reflective
* checks.
* 
* The unnamedModules export all the packages they own and have read access to all modules
* they require access to. 
```

Reply:
Apart from this minor formatting change, this looks good.